### ISSUE TITLE
karma@0.13 compatibility with karma-intellij

### ISSUE THREAD

**Issue by segrey** [karma-runner/karma#1500]
WebStorm 10.0.4 integration fails to run tests using karma v0.13 (https://github.com/karma-runner/karma-intellij/issues/26).
This problem is already fixed in karma-intellij, but, unfortunately, all IDEs released prior to this fix still fail to run tests with karma v0.13, because karma-intellij is bundled with IDE.

A possible solution to make karma integration working in already released IDEs could be to restore backward compatibility in karma v0.13, like this
```
./lib/server.js

// Backward-compatibility with karma-intellij bundled with WebStorm
// to be removed in karma 0.14
Server.start = function (config) {
  new Server(config).start()
}
```

Restoring backward compatibility would be much appreciated. Thanks.

**Comment by Dignifiedquire** [comment]
I can live with that, as long ass we keep a deprecation message warning people that this will go away soon. Are you up for a PR?

**Comment by segrey** [comment]
Thanks. Sure, will create a PR.

**Comment by segrey** [comment]
Created https://github.com/karma-runner/karma/pull/1501
The next karma release with this PR merged will be compatible with WebStorm 10.0.4 integration.
Any plans for the next release?

**Comment by Dignifiedquire** [comment]
Thanks, I hope to get a new release out either today or tomorrow with those changes.
Any chance you could get me a developer key for webstorm? My trial expired some time ago, so I can't test the integration anymore (as usually use emacs for development)

**Comment by segrey** [comment]
WebStorm has free license for open source projects. Please apply [here](https://www.jetbrains.com/buy/opensource/?product=webstorm). Hope it will work smoothly, but in case of any problems I'm here to help.

**Comment by Dignifiedquire** [comment]
Thanks, I didn't know about that.

**Comment by segrey** [comment]
Improved PR https://github.com/karma-runner/karma/pull/1503

**Comment by m69** [comment]
Any updates on the 0.13.3 release?

**Comment by Dignifiedquire** [comment]
Published just now in `0.13.3`

**Comment by segrey** [comment]
Thanks a lot for the quick turnaround!
